packages/compiler/core/checker.ts

```diff
@@ -198,8 +299,44 @@ export function createChecker(program: Program): Checker {
     getMergedNamespace,
     cloneType,
     resolveCompletions,
+    evalProjection: evalProjectionStatement,
+    project,
+    getProjectionInstructions,
+    neverType,
+    errorType,
+    voidType,
+    createType,
+    createAndFinishType,
+    createFunctionType,
+    createLiteralType,
+    finishType,
   };
 
+  const projectionMembers = createProjectionMembers(checker);
+  return checker;
+
+  function initializeCadlIntrinsics() {
+    // a utility function to log strings or numbers
+    cadlNamespaceNode!.exports!.set("log", {
+      kind: "function",
+      name: "log",
+      value(p: Program, str: string): Type {
+        console.log(str);
+        return voidType;
+      },
+    });
+
+    // a utility function to dump a type
+    cadlNamespaceNode!.exports!.set("inspect", {
+      kind: "function",
+      name: "inspect",
+      value(p: Program, str: Type): Type {
+        console.log(str);
```

Comment:
Console.log here left as well

Reply:
The point of the inspect function is to console log the value, though, so I think this one is ok!